companies/shoop.json

```diff
@@ -0,0 +1,17 @@
+{
+    "slug": "shoop",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Shoop Germany GmbH",
+    "address": "Greifswalder Straße 212\n10405 Berlin\nDeutschland",
+    "email": "datenschutz@shoop.de",
```

Comment:
> Do I use the DPO or the generic "datenschutz@" e-mail?

In this case, either is fine. I would probably also lean towards the DPO email whether that's actually better varies from company to company.

> but it's a individual person and this goes against datenanfragens guidelines ("Please make sure that there is no personal data in your record. This means no names in the address and no emails that include names, either.")

That only applies to the data in the actual record. So we couldn't use `kim.mustermensch@company.com` but `kontakt@agidat.de` is fine.

Reply:
The problem isn't the email address, but that I would have to name the DPO in the postal address ("c/o Actual Person, Motzener Straße 25, 12277 Berlin"). So I will leave that as it is now.